wiki/Beatmapping/Overall_Difficulty/en.md

```diff
@@ -0,0 +1,43 @@
+---
+tags:
+- OD
+- Hit Window
+- Notelock
+---
+
+# Overall Difficulty
```

Comment:
Also change folder caps. If you can't change folder caps locally, try `git config core.ignorecase false`.
```suggestion
# Overall difficulty
```

Reply:
on windows i find `git mv` a lot more straightforward, unsetting `core.ignorecase` can potentially make hell of your index if you try to checkout across branches where caps change and your workdir doesnt exactly match what git was expecting

idk maybe not relevant here just warning u lol